frontend/decoder.py

```diff
@@ -13,8 +12,10 @@
     QApplication,
     QAbstractScrollArea,
 )
+from dataclasses import dataclass
 from pydm import Display
 
+sys.path.insert(0, '.')
```

Comment:
I'm trying to get rid of these lines peppered throughout the files because I don't love directly manipulating the python path. Is this necessary for launching from the cavity display or just for launching independently?

Reply:
The path is only in there to launch the decoder independently of the cavity display, so it definitely can be removed and not affect our main cavity display

Reply:
I'll check and see if I can achieve the same thing by changing one of our utils.py file names so that we don't have duplicate file names